src/clr-core/icon-shapes/icon.element.ts

```diff
@@ -55,6 +55,7 @@ export class CdsIcon extends IconMixinClass {
   private _shape: string;
   private _size: string;
 
+  @property({ type: String })
```

Comment:
Found looking at lit docs that the prop has to be on the getter or it can cause unexpected behavior.